### PR TITLE
Fix socrata upload actionworkflow

### PR DESCRIPTION
**Pull request by wrridgeway**
This github workflow is currently failing, this PR is an attempt to fix it.

# BEFORE MERGING
- Remove limits from sql queries
- Change parcel universe asset id to actual open data portal asset id

### CODE REVIEW

**Comment by wrridgeway** [comment]
I'm marking this as ready for review since the workflow and script are now working as anticipated, but I have some questions about it that need to be addressed before it's merged in.

- I had @dfsnow add a `AWS_REGION` variable to the repo - was this superfluous? Is there already a `AWS_DEFAULT_REGION` variable?
- I have our s3 staging directory hard coded because I don't know the variable name for it.
- Do I have superfluous workflow steps?

File: .github/workflows/socrata_upload.yaml [hunk 1]
```diff
@@ -32,27 +32,31 @@ on:
       by_township:
         type: boolean
         description: Chunk Socrata upload by township
-        default: false
         required: false
 
 jobs:
   upload-open-data-assets:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (jeancochrane) [inline .github/workflows/socrata_upload.yaml:39]: **[Question, non-blocking]** What's the story here?

File: .github/workflows/socrata_upload.yaml [hunk 2]
```diff
@@ -62,13 +66,13 @@ jobs:
         env:
           AWS_IAM_ROLE_TO_ASSUME_ARN: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
           AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
-          AWS_ATHENA_S3_STAGING_DIR: ${{ secrets.AWS_ATHENA_S3_STAGING_DIR }}
-          AWS_REGION: ${{ secrets.AWS_REGION }}
+          AWS_ATHENA_S3_STAGING_DIR: ${{ vars.AWS_ATHENA_S3_STAGING_DIR }}
+          AWS_REGION: ${{ vars.AWS_REGION }}
```
Comment (jeancochrane) [inline .github/workflows/socrata_upload.yaml:70]: **[Question, non-blocking]** Elsewhere we just hardcode this to `"us-east-1"`, any reason not to do that here as well?

https://github.com/ccao-data/data-architecture/blob/ac90562f411757e53b74349f2eace76621f81f7d/dbt/scripts/run_iasworld_data_tests.py#L1206-L1210

File: socrata/socrata_upload.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 
 # Connect to Athena
 cursor = connect(
-    s3_staging_dir=str(os.getenv("AWS_ATHENA_S3_STAGING_DIR")) + "/",
+    s3_staging_dir="s3://ccao-athena-results-us-east-1/",
```
Comment (jeancochrane) [inline socrata/socrata_upload.py:15]: **[Question, non-blocking]** If we're not going to read `AWS_ATHENA_S3_STAGING_DIR` from the environment, do we need to pass it in as an env var in the workflow definition above?

**Review by jeancochrane — APPROVED** [review]
A couple questions below, but no serious concerns from my end!

File: .github/workflows/socrata_upload.yaml (hunk 1 above)
Comment (wrridgeway) [reply .github/workflows/socrata_upload.yaml:39]: I was struggling with the github action telling me it couldn't find most versions of python. I came across some advice suggesting that the version of ubuntu was the culprit and it worked. I'll try again with latest and see if this solution is still needed.

File: socrata/socrata_upload.py (hunk 3 above)
Comment (wrridgeway) [reply socrata/socrata_upload.py:15]: I was getting an error that the staging directory wasn't a proper S3 location. My only thought was that perhaps I'm not passing the script an environmental var that actually exists?

File: .github/workflows/socrata_upload.yaml (hunk 1 above)
Comment (wrridgeway) [reply .github/workflows/socrata_upload.yaml]: Okay, seems to no longer be an issue.

File: socrata/socrata_upload.py (hunk 3 above)
Comment (wrridgeway) [reply socrata/socrata_upload.py:15]: @jeancochrane This still causes an error for some reason... to you know if the github var actually exists?

**Pull request merged**